src/common.js

```diff
@@ -75,3 +11,18 @@ export function nodeListForEach (nodes, callback) {
     callback.call(window, nodes[i], i, nodes)
   }
 }
+
+// Used to generate a unique string, allows multiple instances of the component without
+// Them conflicting with each other.
+// https://stackoverflow.com/a/8809472
+export function generateUniqueID () {
```

Comment:
Is there a simpler version of this?

Reply:
It may be a bit of an overkill, but I get is trying to follow [rfc4122](https://www.ietf.org/rfc/rfc4122.txt) so might be ok.